### ISSUE TITLE
Too many logs related to Sass

### ISSUE THREAD

**Issue by rmartins90** [vuejs/vue-jest#69]
https://cl.ly/3U3z3K2K1P3H

**Comment by eddyerburgh** [comment]
Yeah this isn't ideal. 

We can either remove the warning, which could sting some developers, or add sass support.

**Comment by rmartins90** [comment]
There's no way to emit only one warning?

**Comment by eddyerburgh** [comment]
Yes actually it was easy to only log it once. I've added it in https://github.com/vuejs/vue-jest/commit/26e63066d9a09330ca0e82df8a0e6e0a3d59b698 . It will be in the next release. Thanks for bringing this to my attention!

**Comment by ilyaztsv** [comment]
@eddyerburgh Hello!

As I can see [26e6306](https://github.com/vuejs/vue-jest/commit/26e63066d9a09330ca0e82df8a0e6e0a3d59b698) did not resolve the issue.

Please see the example repo: https://github.com/ilyaztsv/vue-jest-warnings

### Reproducing steps:

```
npm i
npm run test
```

**Comment by ilyaztsv** [comment]
can be closed because https://github.com/vuejs/vue-jest/issues/75#issuecomment-379664086
